src/components/Send.vue

```diff
@@ -80,21 +81,24 @@ export default {
     FormInformation,
   },
   data() {
-    const integerNanoWit = (rule, value, callback) => {
-      const isNanoWit = this.currency === WIT_UNIT.NANO
-      if (isNanoWit && !Number.isInteger(Number(value))) {
-        callback(new Error('Only integer nanoWits values allowed'))
+    const maxNumber = (rule, value, callback) => {
+      const maxNumber = 18446744073709551615
```

Comment:
Since this number is greater than max safe integer, we must use a string to represent it. And since the next line does `if number >= maxNumber`, this maxNumber needs to be exactly `2**64`. Also, add a comment explaining what's this number and why is this the maximum.

```suggestion
      // 2**64. This is the max value of the u64 type in the wallet.
      const maxNumber = "18446744073709551616";
```